glfw/src/Cursor.zig

```diff
@@ -74,9 +80,15 @@ pub inline fn create(image: Image, xhot: isize, yhot: isize) Error!Cursor {
 /// @thread_safety This function must only be called from the main thread.
 ///
 /// see also: cursor_object, glfwCreateCursor
-pub inline fn createStandard(shape: Shape) Error!Cursor {
+pub inline fn createStandard(shape: Shape) error{ InvalidEnum, PlatformError }!Cursor {
```

Comment:
Would you mind breaking out the change for error denormalization (`Error` -> `error{ InvalidEnum, PlatformError }`) into a separate PR?

Actually, before that, it might be worth discussing if this is a good idea or not. I have a few concerns with this:

1. GLFW can change the errors functions return at any moment, we'll have to reflect that and it'll have ABI-breaking consequences for us but not for GLFW.
2. GLFW internally is not always consistent with the error values it says it returns in its documentation, unfortunately. Some functions can return errors that are not documented.

Reply:
Sorry, like I said, a lot of this was done mechanically to just get a working version up and passing. I'll work some regex magic when I get back on my laptop.

Reply:
BTW, if you like regexp find-and-replace for code, you might like this tool from my PL PhD friend: https://comby.dev - it's a bit nicer than regexp for doing code replacement operations

Reply:
That looks severely interesting, and quite useful. Thanks for the recommendation. I used my usual regex to revert the error denormalization, but I'll definitely be looking to try and use comby some time in the very near future.